### DISCUSSION TITLE
chart into view

### DISCUSSION THREAD

**Discussion by PaoloPetra** [iTwin/itwinjs-core#3562] [Q&A] [answered]
Hello everyone good morning,

I should implement graphs for my app by reading Bim data
In the meantime, I wanted to know if itwin makes charts available, and otherwise if it is possible to use very popular libraries such as
react-chartjs-2, or if you have any advice to give me about it since I have not found anything among the various examples made available by you

Thanks for your invaluable help

**Comment by pmconne** [comment, marked as answer]
There is no charting functionality built into iTwin.js. You are welcome to use whatever particular charting library you like to visualize data obtained from an iTwin. I am not aware of any examples.

**Reply by williamkbentley** [reply]
[I made a sandbox that demonstrates something similar](https://www.itwinjs.org/sandboxes/williamkbentley/Resize-Widget-Handler). The sandbox shows a static image. Previous to creating the sandbox I had a similar application working locally that used [plotly.js](https://plotly.com/javascript/) to show the chart. There are dozens of similar packages that you can use, and should work, as @pmconne mentioned.
